packages/starlight/translations/pt.json

```diff
@@ -18,6 +18,7 @@
   "page.lastUpdated": "Última atualização:",
   "page.previousLink": "Anterior",
   "page.nextLink": "Próximo",
+  "page.draft": "This content is a draft and will not be included in production builds.",
```

Comment:
```suggestion
  "page.draft": "Esse conteúdo é um rascunho e não será incluído em builds de produção.",
```

Reply:
Thanks for adding the translation ahead of time, super appreciated :raised_hands: (note that I may ping you if the initial text gets modified before the release)